internal/graph/resolver.go

```diff
@@ -19,31 +19,31 @@ type Resolver struct {
 	*repositories.Repository
 }
 
-func getFloatArgs(ctx context.Context, obj *model.SignalsWithID, agg model.FloatAggregation) (repositories.FloatSignalArgs, error) {
+func getFloatArgs(ctx context.Context, obj *model.SignalsWithID, agg model.FloatAggregation) (*repositories.FloatSignalArgs, error) {
```

Comment:
This is not the point of this PR, but I wonder if we could avoid doing this context stuff using [extra fields](https://gqlgen.com/recipes/extra_fields/).

Reply:
Oh, you're using your own model. Can't it still work?

Reply:
Ah, you want the name.

Reply:
I don't think extra fields solves this problem. The reason we have to do this stuff is because there is currently no elegant way to get arguments from a parent resolver (signals/signalsLatest) in the child resolvers (speed/powerTrainType) . https://github.com/99designs/gqlgen/issues/813

Reply:
People are crapping on [this guy's comment](https://github.com/99designs/gqlgen/issues/813#issuecomment-517601986) but I don't see why it's bad.

Reply:
Like I would stuff the times and stuff into the bigger struct [here](https://github.com/DIMO-Network/telemetry-api/blob/da68fb4c14521cdb1d72ca00927ff8f432664bbb/internal/graph/base.resolvers.go#L16) here; and maybe put the ClickHouse column name in the generated code so you don’t have to run around grabbing that.

Does it work? Is it an improvement? I don’t know.

Reply:
Anyway, something for another time.